files/angular-rickshaw/update.json

```diff
@@ -0,0 +1,10 @@
+{
+  "packageManager": "github",
+  "name": "angular-rickshaw",
+  "repo": "ngyewch/angular-rickshaw",
+  "files": {
+    "basePath": "/",
+    "include": ["README.MD", "LICENSE.html","bower.json","rickshaw.js"],
```

Comment:
None of these files is needed in production. Can you add a minified version to your repo?